insights/combiners/selinux.py

```diff
@@ -43,7 +40,7 @@
 
 
 @combiner(requires=[SEStatus, SelinuxConfig],
-          optional=[GrubConfig, Grub1Config, Grub2Config, Grub2EFIConfig])
+          optional=[Grub1Config, Grub2EFIConfig, Grub2Config, Grub2EFIConfig])
```

Comment:
Should be `Grub1Config, Grub1EFIConfig, Grub2Config, Grub2EFIConfig` ?

Reply:
Oh... my :x: :bangbang:, will fix it